pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -676,9 +676,9 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(pod *v1.Pod, podStatus *ku
 			} else {
 				podCpuResources.CPUShares = podResources.CPUShares
 			}
-			err = pcm.SetPodCgroupConfig(pod, rName, podCpuResources)
+			err = pcm.SetPodCgroupConfig(pod, podCpuResources)
```

Comment:
We're always passing CPUPeriod here, which will overwrite the quota regardless of whether CPU quota is set: https://github.com/opencontainers/runc/blob/main/libcontainer/cgroups/fs2/cpu.go#L55-L75